### PR TITLE
Support for logback-access 2.0.0

### PR DESCRIPTION
**Pull request by sondemar**
closes https://github.com/logfellow/logstash-logback-encoder/issues/1014

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,13 +19,14 @@
...
     <properties>
-        <java.version>8</java.version>
+        <java.version>11</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- runtime dependencies -->
         <jackson.version>2.15.2</jackson.version>
         <java-uuid-generator.version>4.2.0</java-uuid-generator.version>
-        <logback.version>1.3.7</logback.version>
+        <logback.core.version>1.5.3</logback.core.version>
```
Comment (runeflobakk) [inline pom.xml:28]:
```suggestion
        <logback.core.version>1.5.6</logback.core.version>
```

**Comment by runeflobakk** [comment]
This is great work, and I hope this can be merged and released soon 🙏 
Logstash Logback Encoder is currently not compatible with Spring Boot 3.3.x and Jetty 12.

File: pom.xml [hunk 2]
```diff
@@ -534,9 +535,9 @@
                         <!-- Setup links to external api docs
                          -->
                         <links>   
-                            <link>https://javadoc.io/doc/ch.qos.logback/logback-core/${logback.version}</link>
-                            <link>https://javadoc.io/doc/ch.qos.logback/logback-classic/${logback.version}</link>
-                            <link>https://javadoc.io/doc/ch.qos.logback/logback-access/${logback.version}</link>
+                            <link>https://javadoc.io/doc/ch.qos.logback/logback-core/${logback.core.version}</link>
+                            <link>https://javadoc.io/doc/ch.qos.logback/logback-classic/${logback.core.version}</link>
+                            <link>https://javadoc.io/doc/ch.qos.logback/logback-access/${logback.access.version}</link>
```
Comment (shmyer) [inline pom.xml:540]: I think the package group artifact id are wrong here. Shouldn't that be:

`<link>https://javadoc.io/doc/ch.qos.logback.access/common/${logback.access.version}</link>`

File: pom.xml (hunk 1 above)
Comment (sondemar) [reply pom.xml]: updated 🙇

File: pom.xml (hunk 2 above)
Comment (sondemar) [reply pom.xml]: good point. corrected 🙇

File: pom.xml [hunk 3]
```diff
@@ -19,13 +19,14 @@
...
-        <java.version>8</java.version>
+        <java.version>11</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- runtime dependencies -->
         <jackson.version>2.15.2</jackson.version>
         <java-uuid-generator.version>4.2.0</java-uuid-generator.version>
-        <logback.version>1.3.7</logback.version>
+        <logback.core.version>1.5.6</logback.core.version>
+        <logback.access.version>2.0.1</logback.access.version>
```
Comment (reneleonhardt) [inline pom.xml:29]:
```suggestion
        <logback.access.version>2.0.2</logback.access.version>
```

File: pom.xml [hunk 4]
```diff
@@ -19,13 +19,14 @@
     </scm>
 
     <properties>
-        <java.version>8</java.version>
+        <java.version>11</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- runtime dependencies -->
         <jackson.version>2.15.2</jackson.version>
```
Comment (reneleonhardt) [inline pom.xml:26]:
```suggestion
        <jackson.version>2.15.4</jackson.version>
```

**Pull request merged**